### ISSUE TITLE
Metadata key is vulnerable to long keys from an input Map.

### ISSUE THREAD

**Issue by danbills** [broadinstitute/cromwell#2853]
A user in the forum was using particularly long filenames as keys to a map, causing his workflow to fail upon flushing metadata to the DB.

He was able to use a workaround of two maps having common keys, one having the filename and the other having the original value of his original map.

--
Original Forum post:
https://gatkforums.broadinstitute.org/gatk/discussion/comment/43927#Comment_43927

**Comment by katevoss** [comment]
@danbills is the long filename limitation in our docs?

**Comment by danbills** [comment]
@katevoss No, but it's an edge case and we don't really have WDL-specific docs so far as I can tell.  In other words I'm not sure it's worth communicating and I don't see a good way of communicating it.
